### DISCUSSION TITLE
TerriaMap - Branding - logo and browser tab text

### DISCUSSION THREAD

**Discussion by mattpgreen** [TerriaJS/terriajs#7177] [General] [answered]
Hi, this is my 1st entry here.  I have set up a standard TerriaMap instance in AWS ec2.  There is strange behaviour when I try to change the title in the browser tab. I have edited the 'appName" value in config but it is not changing the text shown in the browser tab for the map.  I have also edited index.ejs. I'm guessing there is a hard code somewhere else that is controlling this - perhaps "version".  Can someone please help early this week - I was hoping to show progress on this matter tomorrow, Tuesday - Darwin, Australia time. Thanks, Matt

**Comment by na9da** [comment, marked as answer]
@mattpgreen - sorry if this is too late but once you change the title in the [index.ejs](https://github.com/TerriaJS/TerriaMap/blob/e5d79829d41101f95231c99fc1a3d4644e177f25/wwwroot/index.ejs#L30) template file, you should rebuild the app. 

Ideally i think it should set the title from the `appName` but that's not happening right now.
